scripts/dockerrun.sh

```diff
@@ -115,7 +115,7 @@ execute() {
 echo "Building Docker Container using Dockerfile: $DOCKERFILE"
 
 # Get the name of Docker image.
-image=$(grep -i "^FROM " "$DOCKERFILE" | awk '{ print $2 }')
+image=$(grep -i "^FROM " "$DOCKERFILE/Dockerfile" | awk '{ print $2 }')
```

Comment:
`DOCKERFILE` should point to the directory containing 'Dockerfile'. Else the following command will fail.  
`docker build --build-arg USER_ID=$(id -u) -t $DOTNET_BUILD_CONTAINER_TAG $DOCKERFILE`
If we want to pass a file then `-t` should be replaced with `-f`